pom.xml

```diff
@@ -63,7 +63,7 @@
     </distributionManagement>
 
     <properties>
-        <aws.java.sdk.version>2.15.51</aws.java.sdk.version>
+        <aws.java.sdk.version>2.15.79</aws.java.sdk.version>
```

Comment:
Is .79 the absolute minimum version?

Reply:
Yup, here is the [change log](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md#21579-2021-02-09) for 2.15.79